src/many/src/types/identity/cose.rs

```diff
@@ -88,23 +88,13 @@ impl CoseKeyIdentity {
         trace!("Creating NIST P-256 SEC1 encoded point");
         let points = p256::EncodedPoint::from_bytes(raw_points).map_err(|e| e.to_string())?;
 
-        let cose_key = CoseKey {
-            kty: KeyType::Assigned(coset::iana::KeyType::EC2),
-            alg: Some(Algorithm::Assigned(coset::iana::Algorithm::ES256)),
-            key_ops: BTreeSet::from([KeyOperation::Assigned(coset::iana::KeyOperation::Verify)]),
-            params: vec![
-                (
-                    Label::Int(coset::iana::Ec2KeyParameter::X as i64),
-                    Value::Bytes(points.x().unwrap().to_vec()),
-                ),
-                (
-                    Label::Int(coset::iana::Ec2KeyParameter::Y as i64),
-                    Value::Bytes(points.y().unwrap().to_vec()),
-                ),
-            ],
-            ..Default::default()
-        };
-
+        let cose_key = ecdsa_cose_key(
+            (
+                Some(Value::Bytes(points.x().unwrap().to_vec())),
```

Comment:
Suggestion; I'm curious if it makes sense to have the `e[cd]dsa_cose_key` signature takes `Vec<u8>` instead of `Value`? It would simplify usage everywhere. It could even be `EcPoint` or whatever the type is from Coset.

Reply:
It simplifies key creation code but "complexifies" `public_key()` since we already have a `Value`. 

WDYT?

I'm also not sure to understand what you mean by

> It could even be EcPoint or whatever the type is from Coset.

Reply:
> I'm also not sure to understand what you mean by
>
> > It could even be EcPoint or whatever the type is from Coset.

I mean the type returned by `points.x()` and `points.y()`.

---

I'm okay either way. I just don't like having internal serialized types transcend API boundaries. It means that anyone using `ecdsa` needs to use Ciborium and understand it, while it should really be an encapsulated impl detail. But it's not blocking so LGTM.